Providers/Pocket/PocketFeed.swift

```diff
@@ -76,24 +36,33 @@ class Pocket: FeatureFlaggable, URLCaching {
     }()
 
     // Fetch items from the global pocket feed
-    func globalFeed(items: Int = 2) -> Deferred<Array<PocketStory>> {
+    func globalFeed(items: Int = 2) -> Deferred<[PocketFeedStory]> {
         if shouldUseMockData {
-            return getMockDataFeed(items: items)
+            return getMockDataFeed(count: items)
         } else {
             return getGlobalFeed(items: items)
         }
     }
 
-    private func getGlobalFeed(items: Int = 2) -> Deferred<Array<PocketStory>> {
-        let deferred = Deferred<Array<PocketStory>>()
+    // Fetch items from the global pocket feed
+    func sponsoredFeed(items: Int = 2) -> Deferred<[PocketSponsoredStory]> {
+        if shouldUseMockData {
+            return getMockSponsoredFeed()
+        } else {
+            return Deferred(value: [])
+        }
```

Comment:
Just to understand, there will be another story to integrate the new API in this class? But at the moment we'll use only the mock data?

Reply:
Yes. Mocking data was used only to show the UI. I'm going to add in a separate PR the API integration similar with the work done on 'Contile'